src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -1123,7 +1123,7 @@ public RavenConfiguration CreateDatabaseConfiguration(StringSegment databaseName
                 databaseIsBeenDeleted == false)
                 throw new DatabaseNotRelevantException(databaseName + " is not relevant for " + _serverStore.NodeTag);
 
-            return CreateDatabaseConfiguration(_serverStore, databaseRecord.DatabaseName, databaseRecord.Settings);
+            return CreateDatabaseConfiguration(_serverStore, databaseName.Value, databaseRecord.Settings);
```

Comment:
shouldn't we use the databaseRecord.DatabaseName ? It was the original one, not sure if casing is correct in the segment there?

Reply:
it is originated from the `RawDatabaseRecord.DatabaseName`, so the casing should be ok.